src/domain/app/__tests__/apolloRHHC.test.tsx

```diff
@@ -39,7 +38,7 @@ const mocks = [
 describe('test issues in connection between the apollo client and the RHHC-lib', () => {
   describe('apollo client should work with RHHC-lib', () => {
     it('has apollo client initialized properly in the config', () => {
-      const { result } = customRenderHook(() => useConfig());
+      const { result } = renderHook(() => useConfig());
```

Comment:
There was no custom renderHook in testUtils, it was the exported renderHook from @testing-library/react. So that's why I have changed customRenderHook to be used directly as renderHook from @testing-library/react.